adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala

```diff
@@ -109,6 +114,28 @@ private object FeatureOrdering extends FeatureOrdering[Feature] {}
 object FeatureRDD {
 
   /**
+   * A GenomicRDD that wraps an RDD of Feature data.
+   *
+   * @param rdd An RDD of genomic Features.
+   * @param sequences The reference genome this data is aligned to.
+   */
+  def apply(rdd: RDD[Feature],
+            sequences: SequenceDictionary): FeatureRDD = {
+    new RDDBoundFeatureRDD(rdd, sequences)
```

Comment:
It's equivalent to call either `new` or the `apply` method. If you have a strong preference, I'll switch over.